### PR TITLE
Add new version of `exceptionite` to improve debugging experience

### PR DESCRIPTION
**Pull request by girardinsamuel**
We should wait for https://github.com/MasoniteFramework/exceptionite/pull/11 to be merged and `exceptionite==2.0.0` to be deployed.

### CODE REVIEW

File: src/masonite/exceptions/ExceptionHandler.py [hunk 1]
```diff
@@ -62,23 +60,7 @@ def handle(self, exception):
...
-                },
-                "Headers": request.header_bag.to_dict(),
-            }
-        )
-
-        return response.view(handler.render(), status=500)
+        exceptionite = self.get_driver("exceptionite")
+        exceptionite.start(exception)
+        exceptionite.render("terminal")
+        return response.view(exceptionite.render("javascript"), status=500)
```
Comment (girardinsamuel) [inline src/masonite/exceptions/ExceptionHandler.py]: you can see the renderer used here

**Comment by josephmancuso** [comment]
Screen Shot 2022-02-07 at 10 03 52 PM

This says upgrade but its really a downgrade

**Comment by josephmancuso** [comment]
Screen Shot 2022-02-07 at 9 58 00 PM

Hovering over these options make the text color same as background color

**Comment by josephmancuso** [comment]
Screen Shot 2022-02-07 at 10 06 02 PM

I think we have to do a replace(`<br>`) on these stack traces if the language is python

**Comment by girardinsamuel** [comment]
> Screen Shot 2022-02-07 at 10 03 52 PM
> 
> This says upgrade but its really a downgrade

Yep for this one it's a bit special. It's because I have installed a local version which is not released on PyPI yet..

**Comment by girardinsamuel** [comment]
For the other issues, I will take a look, that's some of the things left to do 😉

File: src/masonite/exceptions/ExceptionHandler.py [hunk 2]
```diff
@@ -62,23 +60,7 @@ def handle(self, exception):
...
-                    "Input": request.input_bag.all_as_values() or None,
-                    # 'Parameters': request.url_params,
-                    "Request Method": request.get_request_method(),
-                },
-                "Headers": request.header_bag.to_dict(),
-            }
-        )
-
-        return response.view(handler.render(), status=500)
+        exceptionite = self.get_driver("exceptionite")
```
Comment (girardinsamuel) [inline src/masonite/exceptions/ExceptionHandler.py]: now it's more clear here

File: requirements.txt [hunk 3]
```diff
@@ -4,7 +4,8 @@ black
 cleo>=0.8.1,<0.9
 cryptography>=3.3.1,<4.0
 dotty_dict>=1.3.0<1.40
-exceptionite>=1.0,<1.1
+# exceptionite>=2.0,<3
+https://github.com/girardinsamuel/exceptionite/archive/refs/heads/2.0-dev.zip
```
Comment (girardinsamuel) [inline requirements.txt:8]: once exceptionite is officialy release we switch those lines and then the PR should still pass and we can merge !

**Comment by girardinsamuel** [comment]
@josephmancuso ready for review !

**Review by josephmancuso — APPROVED** [review]

**Pull request merged**